packages/happy-dom/src/event/events/PointerEvent.ts

```diff
@@ -6,37 +6,46 @@ import IPointerEventInit from './IPointerEventInit.js';
  */
 export default class PointerEvent extends MouseEvent {
 	public readonly pointerId: number = 0;
-	public readonly width: number = 0;
-	public readonly height: number = 0;
+	public readonly width: number = 1;
+	public readonly height: number = 1;
 	public readonly pressure: number = 0;
 	public readonly tangentialPressure: number = 0;
 	public readonly tiltX: number = 0;
 	public readonly tiltY: number = 0;
 	public readonly twist: number = 0;
+	public readonly altitudeAngle: number = 0;
+	public readonly azimuthAngle?: number = 0;
 	public readonly pointerType: string = '';
 	public readonly isPrimary: boolean = false;
+	public readonly coalescedEvents: PointerEvent[] = [];
+	public readonly predictedEvents: PointerEvent[] = [];
 
 	/**
 	 * Constructor.
 	 *
 	 * @param type Event type.
 	 * @param [eventInit] Event init.
 	 */
-	constructor(type: string, eventInit: IPointerEventInit = null) {
+	constructor(type: string, eventInit: IPointerEventInit = {}) {
```

Comment:
https://w3c.github.io/pointerevents/#dom-pointerevent-constructor

> constructor([DOMString](https://webidl.spec.whatwg.org/#idl-DOMString) type, optional [PointerEventInit](https://w3c.github.io/pointerevents/#dom-pointereventinit) eventInitDict = **{}**);

Reply:
The downside is that we will get multiple one liner if statements as `eventInit` can be null (see other comment) (e.g. `eventInit?.pointerId ?? 0`), but the benefit is that we will have all default values in one place of course. I guess it is fine.

Reply:
The downside is that we will get multiple one liner if statements as `eventInit` can be null (see other comment) (e.g. `eventInit?.pointerId ?? 0`), but the benefit is that we will have all default values in one place of course, so I think you are right that it is better to go with that.